app/wizards/placements/add_placement_wizard/mentors_step.rb

```diff
@@ -4,15 +4,23 @@ class Placements::AddPlacementWizard::MentorsStep < Placements::BaseStep
   validates :mentor_ids, presence: true
 
   def mentors_for_selection
-    school.mentors
+    school_mentors.order_by_full_name
   end
 
   def mentor_ids=(value)
     super normalised_mentor_ids(value)
   end
 
-  def wizard_attributes
-    { mentor_ids: mentor_ids - NOT_KNOWN }
+  def mentors
+    return [] if mentor_ids == NOT_KNOWN
```

Comment:
Consider using:
```suggestion
    return Placements::Mentor.none if mentor_ids == NOT_KNOWN
```
https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/none
This way, you can still use `mentors.pluck(:trn)` and it won't error because it's an array.

Reply:
Done! Thanks.